fdm-app/app/components/blocks/atlas-fields/groundwater.tsx

```
@@ -0,0 +1,101 @@
+export function GroundWaterSkeleton() {
+    return (
+        <Card>
+            <CardHeader>
+                <CardTitle>Grondwater</CardTitle>
+                <CardDescription>
+                    De geschatte grondwaterstanden voor dit perceel.
+                </CardDescription>
+            </CardHeader>
+            <CardContent className="grid grid-cols-3 gap-4">
+                {[...Array(3)].map((_, index) => (
+                    <Card key={index}>
```

Biome (2.1.2)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)